policy-controller/k8s/status/src/index.rs

```diff
@@ -74,14 +106,41 @@ impl Index {
         updates: UnboundedSender<Update>,
     ) -> SharedIndex {
         Arc::new(RwLock::new(Self {
-            _name: name.to_string(),
-            _claims: claims,
+            name: name.to_string(),
+            claims,
             updates,
             http_routes: HashMap::new(),
             servers: HashSet::new(),
         }))
     }
 
+    pub async fn run(index: Arc<RwLock<Self>>) {
```

Comment:
perhaps worth a quick comment here about what `run` does (triggers reconciliation every 10 seconds or whenever we acquire the lease)